sklearn/ensemble/_forest.py

```diff
@@ -615,6 +615,13 @@ def feature_importances_(self):
     )
     @property
     def n_features_(self):
+        """Number of features when fitting the estimator.
+
+        Returns
+        -------
+        n_features_in_ : int
+            The number of features when fitting the estimator.
+        """
```

Comment:
The ending underscore causes some issue with sphinx that understand it as an hyperlink. I think that we can get around and have a more condense way to write the docstring and being numpydoc compliant:

```suggestion
        """int: Number of features when fitting the estimator."""
```

Reply:
Thanks, @glemaitre. It matches the suggestion by sphinx for properties, but numpydoc was raising GL02. Added a line break for that and it passes, but `black` will remove it.  One way is to add text after the attribute definition. Any suggestion for the additional text?

```suggestion
        """int: Number of features when fitting the estimator.        
        Some additional text.
        """
```

Reply:
We are going to skip the GL02 I think: https://github.com/scikit-learn/scikit-learn/pull/20473
So whenever the other PR is merged, you can merge `main` into your branch and it will be OK to have an inline docstring.